foundry.toml

```diff
@@ -6,6 +6,12 @@ libs = ['lib']
 solc = "0.8.17"
 fs_permissions = [{ access = "read", path = "./broadcast"}]
 gas_reports = ["*"]
+no_match_test = "SLOW"
+
+[invariant]
+fail_on_revert = false
+# depth = 32
+# runs = 1
```

Comment:
They're useful to run for invariant tests to get more in depth checks, it's nice to just uncomment rather than look up the keys in case you forget `depth` and `runs`. I don't see any harm in keeping them.